index.bs

```diff
@@ -831,6 +834,14 @@ spec:css-syntax-3;
     1.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
         ancestors=], and `false` otherwise.
 
+    1.  If |options|[{{CredentialRequestOptions/publicKey}}] [=exists=] then
+        if |settings|' [=responsible document=] is **not** [=allowed to use=] the
+        <a data-lt="publickey-credentials-get-feature">publickey-credentials-get</a>
```

Comment:
There's a [shorthand](https://tabatkins.github.io/bikeshed/#autolink-inside) for this, but you don't have to use it:

```suggestion
        [=publickey-credentials-get-feature|publickey-credentials-get=]
```